app/models/vm_or_template/operations/relocation.rb

```diff
@@ -1,4 +1,14 @@
 module VmOrTemplate::Operations::Relocation
+  included do
```

Comment:
You need `extend ActiveSupport::Concern` to use `included` here

Reply:
ah - thank you!